Doc/library/token.rst

```diff
@@ -129,9 +127,11 @@ the :mod:`tokenize` module.
 
 
 .. versionchanged:: 3.5
-   Added :data:`AWAIT` and :data:`ASYNC` tokens. Starting with
-   Python 3.7, "async" and "await" will be tokenized as :data:`NAME`
-   tokens, and :data:`AWAIT` and :data:`ASYNC` will be removed.
+   Added :data:`AWAIT` and :data:`ASYNC` tokens.
 
 .. versionchanged:: 3.7
    Added :data:`COMMENT`, :data:`NL` and :data:`ENCODING` tokens.
+
+.. versionchanged:: 3.7
+   Removed :data:`AWAIT` and :data:`ASYNC` tokens. "async" and "await" are
+   now tokenized as :data:`NAME` tokens.
```

Comment:
Why not saying "keywords". These names are added to keywords.py and it's the PR title, no?